src/components/card/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { CSSProperties } from "react";
+import logo from "../../assets/iterable_logo.png";
+
+interface CardProps {
+  imgSrc?: string;
+  title: string;
+  text: string;
+  primaryBtnLabel?: string;
+  secondaryBtnLabel?: string;
+  disablePrimaryBtn?: boolean;
+  disableSecondaryBtn?: boolean;
+  onClickPrimaryBtn?: () => void;
+  onClickSecondaryBtn?: () => void;
+  imgStyle?: CSSProperties;
+  titleStyle?: CSSProperties;
+  cardStyle?: CSSProperties;
+  textStyle?: CSSProperties;
+  primaryBtnStyle?: CSSProperties;
+  secondaryBtnStyle?: CSSProperties;
+}
+
+const Card = (props: CardProps) => {
```

Comment:
would prefer to keep this consistent with the other components and destructure the props either inline in the component declaration or just below.